code.json

```diff
@@ -58,14 +58,14 @@
       "licenses": [
         {
           "name": "Public Domain, CC0-1.0",
-          "URL": "https://code.usgs.gov/astrogeology/ale/-/raw/main/LICENSE.md"
+          "URL": "https://code.usgs.gov/astrogeology/ale/-/raw/0.9/LICENSE.md"
```

Comment:
0.9.0, not 0.9

Reply:
So the branch will be 0.9, with bugfix versions being tags off of the 0.9 branch

Reply:
I'm saying, I think you need the bug fix '.0' at the end of the tag. Unless you want to have:
- 0.8.8
- 0.9
- 0.9.1
- 0.9.2
- 0.9.x
- 0.10
- 0.10.1

Reply:
The docker code check will fail here. You have to be consistent with the "version":"0.9.0" and the URLS in line 61, 67, 68. They all have to be the same version number (0.9 or 0.9.0) - no mixing).